### PR TITLE
docs: update rereject leader docs

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: multi-data-centers-in-one-city-deployment.md [hunk 1]
```diff
@@ -71,6 +71,9 @@ member leader_priority pdName2 4
 member leader_priority pdName3 3
 ```
 
+> **注意**
+> 5.2 以上版本已不支持，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (rleungx) [inline multi-data-centers-in-one-city-deployment.md:75]: I think we still support it but the label-scheduler is not added by default.

File: config-templates/geo-redundancy-deployment.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ server_configs:
     replication.location-labels: ["zone","dc","rack","host"]
     replication.max-replicas: 5
     label-property:
-      reject-leader:
+      reject-leader: # v5.2 以上版本请使用 Placement Rules 代替
```
Comment (TomShawn) [inline config-templates/geo-redundancy-deployment.yaml:33]:
```suggestion
      reject-leader: # v5.2 及以上版本请使用 Placement Rules 代替
```

File: geo-distributed-deployment-topology.md [hunk 3]
```diff
@@ -82,9 +82,12 @@ aliases: ['/docs-cn/dev/geo-distributed-deployment-topology/']
               value: "sha"
     ```
 
+    > **注意**
+    > 5.2 以上版本已不支持，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline geo-distributed-deployment-topology.md:86]: 能否说明不支持什么？

File: multi-data-centers-in-one-city-deployment.md (hunk 1 above)
Comment (TomShawn) [reply multi-data-centers-in-one-city-deployment.md:75]:
```suggestion
> TiDB 5.2 及以上版本不支持通过以上模板设置调度策略。若要设置调度策略，请使用 [Placement Rules](/configure-placement-rules.md)。

```

File: geo-distributed-deployment-topology.md (hunk 3 above)
Comment (TomShawn) [reply geo-distributed-deployment-topology.md:86]:
```suggestion
    > TiDB 5.2 及以上版本不支持通过以上 YAML 模板设置调度策略。若要设置调度策略，请使用 [Placement Rules](/configure-placement-rules.md)。
```

File: three-data-centers-in-two-cities-deployment.md [hunk 4]
```diff
@@ -192,6 +192,9 @@ tikv_servers:
     ```
     config set label-property reject-leader dc 3
     ```
+  
+    > **注意**
+    > 5.2 以上版本已不支持，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline three-data-centers-in-two-cities-deployment.md:197]:
```suggestion
    > TiDB 5.2 及以上版本不支持通过以上配置来禁用异地中心的 Raft leader 的调度。请使用 [Placement Rules](/configure-placement-rules.md) 替代该配置。
```

File: geo-distributed-deployment-topology.md [hunk 5]
```diff
@@ -82,9 +82,12 @@ aliases: ['/docs-cn/dev/geo-distributed-deployment-topology/']
               value: "sha"
     ```
 
+    > **注意**
+    > TiDB 5.2 及以上版本默认已不开启 `label-property` 支持。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline geo-distributed-deployment-topology.md:86]:
```suggestion
    > **注意：**
    >
    > TiDB 5.2 及以上版本默认不支持 `label-property` 配置。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md)。
```

File: multi-data-centers-in-one-city-deployment.md [hunk 6]
```diff
@@ -71,6 +71,9 @@ member leader_priority pdName2 4
 member leader_priority pdName3 3
 ```
 
+> **注意**
+> TiDB 5.2 及以上版本默认已不开启 `label-property` 支持。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline multi-data-centers-in-one-city-deployment.md:75]:
```suggestion
> **注意：**
>
> TiDB 5.2 及以上版本默认不支持 `label-property` 配置。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md)。
```

File: three-data-centers-in-two-cities-deployment.md [hunk 7]
```diff
@@ -192,6 +192,9 @@ tikv_servers:
     ```
     config set label-property reject-leader dc 3
     ```
+  
+    > **注意**
+    > TiDB 5.2 及以上版本默认已不开启 `label-property` 支持。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline three-data-centers-in-two-cities-deployment.md:197]:
```suggestion
    > **注意：**
    >
    > TiDB 5.2 及以上版本默认不支持 `label-property` 配置。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md)。
```

File: config-templates/geo-redundancy-deployment.yaml [hunk 8]
```diff
@@ -29,7 +29,7 @@ server_configs:
   pd:
     replication.location-labels: ["zone","dc","rack","host"]
     replication.max-replicas: 5
-    label-property:
+    label-property: # TiDB 5.2 及以上版本默认已不开启 `label-property` 支持。若要设置副本策略，请使用 [Placement Rules](/configure-placement-rules.md) 替代该功能。
```
Comment (TomShawn) [inline config-templates/geo-redundancy-deployment.yaml:32]:
```suggestion
    label-property:  # TiDB 5.2 及以上版本默认不支持 label-property 配置。若要设置副本策略，请使用 Placement Rules。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by rleungx — APPROVED** [review]

**Pull request merged**